common/Log.cpp

```diff
@@ -65,9 +67,28 @@ namespace Log
 
         const std::string& getName() const { return _name; }
 
+        void setLevel(const std::string& logLevel) { _logLevel = logLevel; }
+
+        const std::string& getLevel() const { return _logLevel; }
+
         void setLogger(Poco::Logger* logger) { _logger = logger; };
```

Comment:
How do we avoid leaking that ? Presumably at the end of the DocumentBroker::poll - we could check if this is set, and unset / free / clean it up (?) I guess it's only for debugging but still =)

Anyhow - looks lovely; thanks Tor!

Reply:
There is a FIXME about that: 
 ```
            // FIXME: What to do with the previous thread-local logger, if any? Will deleting it
            // destroy also its channel? That won't be good as we use the same channel for all
            // loggers. Best to just leak it?
```